lib/broken_record/slack_notifier.rb

```diff
@@ -1,27 +1,52 @@
 require 'net/http'
 module BrokenRecord
   class SlackNotifier
-    def initialize(options)
-      @options = BrokenRecord::Config.slack_options.merge(options)
+    def initialize(initial_options)
+      @options = default_options
```

Comment:
I prefer the three lines actually.  It's more lines, but I consider something like this is like **making text bold**, it's clearer what's going on.